system-tests/fix/fix.test.js

```diff
@@ -72,6 +72,21 @@ describe("fix", () => {
       });
   });
 
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip("doesn't strip BOM", () => {
+    return stylelint
+      .lint({
+        files: [stylesheetPath],
+        configFile: systemTestUtils.caseConfig("fix"),
+        fix: true
+      })
+      .then(() => {
+        return pify(fs.readFile)(stylesheetPath, "utf8").then(fileContent => {
+          expect(fileContent.startsWith("\uFEFF")).toBe(true);
+        });
+      });
+  });
+
```

Comment:
Can you add new test case instead modify current

Reply:
By test case you mean a new CSS file?

Reply:
Yes

Reply:
Do I need to create another folder under `system-tests` or the current solution will be appropriate?